probatus/utils/arrayfuncs.py

```diff
@@ -267,11 +267,4 @@ def preprocess_labels(y, y_name=None, index=None, verbose=0):
     # Make sure that y is a series with correct index
     y = assure_pandas_series(y, index=index)
 
-    # Warn if not binary labels
```

Comment:
Instead of removing this i would do the following:
add parameter: allow_regression=False, and then if the parameter is true we skip this part, otherwise it would run.

The reason behind is for other features we will not support regression, and only for shaprfecv for now we will.

Reply:
I can add this. Are you thinking to set it to False by default?

Reply:
Actually let's remove that, the models have their own ways of handling if wrong labels are passed